src/libs/API/parameters/DismissViolationParams.ts

```diff
@@ -1,6 +1,7 @@
 type DismissViolationParams = {
     name: string;
     transactionIDList: string;
+    reportActionIDList?: string;
```

Comment:
```suggestion
    reportActionIDList: string;
```

I don't think we should make this optional if we are always passing the param

Reply:
My thought process was that since this is for every violation we might not want to add a report action for some of them, but I guess we can make it optional when that happens..

Reply:
> but I guess we can make it optional when that happens..

I think that is better